frontend/src/components/MapView.js

```diff
@@ -22,92 +23,132 @@ class MapView extends Component {
       allEvents: [],
       location: undefined,
       plusCode: props.plusCode,
-      showInfoWindows: true
+      showInfoWindows: true,
+      contents: null,
+      resizeState: false,
+      maxWidth: 100
     };
+
+    document.addEventListener('domready', () => {
+      console.log("asdas");
+      document.querySelector('.gm-style-iw').addEventListener('click', this.printSomething);
+    });
+
     var plusCode = this.state.plusCode;
-    console.log(this.state.plusCode);
-    console.log("constr");
     this.queryEventsAndStoreInMemory(plusCode);
+    this.renderInfo = this.renderInfo.bind(this);
   }
 
-  UNSAFE_componentWillReceiveProps(nextProps) {
-    console.log("componentWillRec");
-    console.log(nextProps)
-    this.setState({ showInfoWindows: false, allEvents: [], plusCode: nextProps.plusCode }, () => {
-      console.log(this.state.showInfoWinddows);
-      this.queryEventsAndStoreInMemory(nextProps.plusCode);
-      this.setState({showInfoWindows: true});
+  async UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
I don't know what this function name is all about, but seems unsafe

Reply:
it's a react component